python/geom.py

```diff
@@ -239,6 +239,29 @@ def transform(self, m):
         for s in self.H_susceptibilities:
             s.transform(m)
 
+    def rotate(self, rotations):
+        for rot in rotations:
+            if np.count_nonzero(rot) != 1:
+                raise ValueError("Each rotation vector should only have 1 coordinate.")
```

Comment:
would be more general to give a rotation vector (direction = axis, length = angle).  Formula: https://en.wikipedia.org/wiki/Rotation_matrix#Rotation_matrix_from_axis_and_angle

Reply:
addressed in e28c961